ports/mimxrt/Makefile

```diff
@@ -50,12 +63,12 @@ INC += -I$(BUILD)
 INC += -I$(TOP)
 INC += -I$(TOP)/$(MCU_DIR)
 INC += -I$(TOP)/$(MCU_DIR)/drivers
-INC += -I$(TOP)/$(MCU_DIR)/project_template
 INC += -I$(TOP)/lib/cmsis/inc
 INC += -I$(TOP)/lib/oofatfs
 INC += -I$(TOP)/lib/tinyusb/hw
 INC += -I$(TOP)/lib/tinyusb/hw/bsp/teensy_40
 INC += -I$(TOP)/lib/tinyusb/src
+INC += -Ihal
```

Comment:
please move this up to just below `-I.`, so that current-dir includes are before other ones

Reply:
fixed.